### ISSUE TITLE
Sometimes mutaplasmid.space misdetects if contact was sold or not

### ISSUE THREAD

**Issue by DarkFenX** [StephenSwat/eve-abyssal-market#21]
I will put here known cases when detection shows wrong status:

- https://mutaplasmid.space/module/1029280723727/ - owner is Nick on abyssal discord, claims contract was expired

**Comment by StephenSwat** [comment]
Unfortunately there is no official API to check whether a contract expired or not. The way the website tries to discover this information right now is to generate an error by requesting the item list for an unavailable contract and determining what happened to the contract from the error code. I'll investigate why this strategy is not working for this particular contract but I am fairly certain there is no better way to gather this information unfortunately.

**Comment by DarkFenX** [comment]
Yeah I am not implying that it should be switched, but as we gather more data - maybe we will find a way to automatically detect these errors and fix them.

**Comment by DarkFenX** [comment]
Recently all of my expired contracts are shown as if they were sold.
